query-engine/core/src/schema_builder/input_types/fields/data_input_mapper/update.rs

```diff
@@ -343,3 +345,74 @@ fn composite_upsert_update_input_field(ctx: &mut BuilderContext, cf: &CompositeF
         None
     }
 }
+
+fn composite_update_many_object_type(ctx: &mut BuilderContext, cf: &CompositeFieldRef) -> InputObjectTypeWeakRef {
```

Comment:
Side note: this illustrates perfectly the amount of duplicated boilerplate we need all the time :/